components/bl-double-calendar-component/src/index.js

```diff
@@ -29,8 +31,8 @@ export default function DoubleCalendarComponent({ component, eventHandlers }) {
         />
         :
         <DateRange
-          toDate={ toDate }
-          fromDate={ fromDate }
+          toDate={ normalizeFormat(toDate, dateFormat) }
```

Comment:
```const formatNormalizer = useMemo(()=> date => normalizeFormat(dateFormat, date))```

Change args order and use closures here. Also both ranges could be memoized too.
Contact me if something unclear.